src/view.js

```diff
@@ -0,0 +1,104 @@
+class View {
+  constructor() {
+    this.parentElement = document.querySelector('.glass__result');
+    this.searchBar = document.querySelector('.glass__search-bar');
+    this.smallCircle = document.querySelector('.circle--small');
+    this.message = "What's the weather like today? ☁";
+  }
+
+  getQuery() {
+    const query = this.searchBar.querySelector('.glass__search-input').value;
+
+    this.clearInput();
+    return query;
+  }
+
+  clearInput() {
+    this.searchBar.querySelector('.glass__search-input').value = '';
+  }
+
+  clear() {
+    this.parentElement.innerHTML = '';
+  }
+
+  render(data) {
+    if (!data) return;
+
+    const markup = `
+    <div class="glass__weather">
+      <span>Updated as of:</span>
+      <p><span>${data.date}</span></p><br>
+      <h2>${data.name}, ${data.country}</h2>
+      <div class="glass__temperature">
+          <img src="${
+  data.iconSmall
+}" alt="weather icon" class="glass__weather-icon">
+          <h1>${data.temp}<sup>°${data.unit === 'metric' ? 'C' : 'F'}</sup></h1>
+      </div>
+      <p>${data.description}</p>
+      <div class="glass__details">
+          <p>Feels like: ${data.feelsLike} °${
+  data.unit === 'metric' ? 'C' : 'F'
+}</p>
+          <p>Humidity: ${data.humidity}%</p>
+          <p>Sunrise: ${data.sunrise}</p>
+          <p>Sunset: ${data.sunset}</p>
+      </div>
+      <div class="glass__switch">
+          <div class="glass__select celcius ${
+  data.unit === 'metric' ? 'glass__select--active' : ''
+}" data-unit="metric">C</div>
+          <div class="glass__select fahrenheit ${
+  data.unit === 'imperial' ? 'glass__select--active' : ''
+}" data-unit="imperial">F</div>
+      </div>
+    </div>
+    `;
+
+    // this._bigCircle.style.backgroundImage = `url(${data.flag})`;
```

Comment:
## Remove commented out code
- [ ] It’s advisable these deleted. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))